### DISCUSSION TITLE
SponsorBlock Add option to only autoskip a segment on certain videos.

### DISCUSSION THREAD

**Discussion by Blaziy** [ReVanced/revanced-discussions#993] [General] [answered]
Basically the title, add an option to SponsorBlock to only auto skip segments only on specific videos you chose. (And the option to do the opposite, auto skip always except on chosen ones)

**Comment by KobeW50** [comment]
I may be misunderstanding you, but this already exists in the SponsorBlock settings.

**Reply by Blaziy** [reply]
Sorry if I'm wrong but I was told in the discord it's not a thing and I haven't found it in the settings
https://discord.com/channels/952946952348270622/1128916208012505148/1128922902968930356

**Reply by KobeW50** [reply]
How would you want to set that a video's segments shouldn't be skipped?

**Reply by Blaziy** [reply]
First thought is in the settings have a section to add links to videos (and channels) with each one having all the SponsorBlock options (So the options to Skip Automatically, Show skip button, disable, etc.) for each type of skip (sponsor, promo, non-music, etc.).
These would override the your default YouTube ones and possibly also add a button on videos somewhere if there's a place for it so people don't have to go and paste the links in.

**Reply by KobeW50** [reply, marked as answer]
> First thought is in the settings have a section to add links to videos (and channels) with each one having all the SponsorBlock options (So the options to Skip Automatically, Show skip button, disable, etc.) for each type of skip (sponsor, promo, non-music, etc.).
> These would override the your default YouTube ones and possibly also add a button on videos somewhere if there's a place for it so people don't have to go and paste the links in.

Alright. The channel whitelist has already been [requested](https://github.com/ReVanced/revanced-patches/issues/840), but you can comment on that request or make a new one for adding individual links. However, I highly doubt anyone would make that patch though because it's not useful for most people
